src/sentry_slack/plugin.py

```diff
@@ -140,12 +155,18 @@ def notify(self, notification):
                 })
 
         if self.get_option('include_tags', project):
+            included_tags = self.get_tag_list('included_tag_keys', project)
+            excluded_tags = self.get_tag_list('excluded_tag_keys', project)
             for tag_key, tag_value in self._get_tags(event):
-                fields.append({
-                    'title': tag_key.encode('utf-8'),
-                    'value': tag_value.encode('utf-8'),
-                    'short': True,
-                })
+                key = tag_key.lower()
```

Comment:
Added TagKey.get_standardized_key in https://github.com/getsentry/sentry/commit/32abb6ea7af92f739c30f7d078aed1dbb67a48c6